### PR TITLE
Created Home Page (Feature/#4)

### LINKED ISSUE #4: Create Home Page

### PR DESCRIPTION
**Pull request by yurdosii**

### CODE REVIEW

**Review by Howrran — DISMISSED** [review]

File: ngfg.client/src/components/HomePage/FormItemList.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.forms-list {
+    display: flex;
+    flex-wrap: wrap;
+    flex-flow: row wrap; // flex-direction, -wrap
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/FormItemList.scss]: new line

File: ngfg.client/src/components/HomePage/NewFormItem.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    &:hover {
+        color: $newItemHoverColor;
+        box-shadow: 0.9vmin 0.9vmin 1.5vmin $itemShadowColor;
+
+        cursor: pointer;
+
+        transition: 0.4s;
+    }
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/NewFormItem.scss]: new line

File: ngfg.client/src/components/HomePage/NewFieldItem.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    &:hover {
+        color: $newItemHoverColor;
+        box-shadow: 0.9vmin 0.9vmin 1.5vmin $itemShadowColor;
+
+        cursor: pointer;
+
+        transition: 0.4s;
+    }
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/NewFieldItem.scss]: new line

File: ngfg.client/src/components/HomePage/FormItem.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    &:hover {
+        box-shadow: 0.9vmin 0.9vmin 1.5vmin $itemShadowColor;
+
+        cursor: pointer;
+
+        transition: 0.4s;
+    }
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/FormItem.scss]: new line

File: ngfg.client/src/components/HomePage/FieldItemList.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.fields-list {
+    display: flex;
+    flex-wrap: wrap;
+    flex-flow: row wrap; // flex-direction, -wrap
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/FieldItemList.scss]: new line

File: ngfg.client/src/components/HomePage/FieldItem.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    &:hover {
+        box-shadow: 0.9vmin 0.9vmin 1.5vmin $itemShadowColor;
+
+        cursor: pointer;
+
+        transition: 0.4s;
+    }
+}
```
Comment (Howrran) [inline ngfg.client/src/components/HomePage/FieldItem.scss]: new line

File: ngfg.client/src/components/Header/Header.scss [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+        text-transform: capitalize !important;
+
+        &:hover {
+            background-color: $navbarColor !important;
+
+            transition: 0.4s;
+        }
+    }
+}
```
Comment (Howrran) [inline ngfg.client/src/components/Header/Header.scss]: new line

File: ngfg.client/src/components/HomePage/FormItemList.scss (hunk 1 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/FormItemList.scss]: Fixed

File: ngfg.client/src/components/HomePage/NewFormItem.scss (hunk 2 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/NewFormItem.scss]: Fixed

File: ngfg.client/src/components/HomePage/NewFieldItem.scss (hunk 3 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/NewFieldItem.scss]: Fixed

File: ngfg.client/src/components/HomePage/FormItem.scss (hunk 4 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/FormItem.scss]: Fixed

File: ngfg.client/src/components/HomePage/FieldItemList.scss (hunk 5 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/FieldItemList.scss]: Fixed

File: ngfg.client/src/components/HomePage/FieldItem.scss (hunk 6 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/FieldItem.scss]: Fixed

File: ngfg.client/src/components/Header/Header.scss (hunk 7 above)
Comment (yurdosii) [reply ngfg.client/src/components/Header/Header.scss]: Fixed

**Review by zzeropilot — DISMISSED** [review]

**Review by IvanKostiv — DISMISSED** [review]

File: ngfg.client/src/components/HomePage/HomePage.js [hunk 8]
```diff
@@ -1,12 +1,107 @@
...
+    }
+
+    getData = () => {
+        
+        axios.get(`${API_URL}/${API_VERSION}/forms/`, {
+            withCredentials: true,
+        }).then(res => {
+            const forms = res.data.forms.reverse().slice(0, LIMIT_ITEMS_NUMBER);
+            this.setState({ forms })
+            // console.log(res);
```
Comment (IvanKostiv) [inline ngfg.client/src/components/HomePage/HomePage.js]: Remove console.log

File: ngfg.client/src/components/HomePage/HomePage.js [hunk 9]
```diff
@@ -1,12 +1,107 @@
...
+            
+        })
+
+        axios.get(`${API_URL}/${API_VERSION}/fields/`, {
+            withCredentials: true,
+        }).then(res => {
+            const fields = res.data.fields.reverse().slice(0, LIMIT_ITEMS_NUMBER);
+            this.setState({ fields })
+            // console.log(res);
```
Comment (IvanKostiv) [inline ngfg.client/src/components/HomePage/HomePage.js]: remove console.log

File: ngfg.client/src/components/HomePage/HomePage.js (hunk 8 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/HomePage.js]: Fixed

**Review by IvanKostiv — APPROVED** [review]

File: ngfg.client/src/components/HomePage/HomePage.js (hunk 9 above)
Comment (yurdosii) [reply ngfg.client/src/components/HomePage/HomePage.js]: Fixed

**Review by dtymkiv — APPROVED** [review]

**Review by zzeropilot — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Pull request merged**